### PR TITLE
Frontend web development react angular vue : History Mode in routes (Vue.js)

### LINKED ISSUE #7634: FRONTEND WITH FRAMEWORK: VueJS-7.5 => History Mode in routes
### Description

Contributors are required to explain about History Mode in routes in VueJs with the "vue-router" plugin, which should cover questions like:-
What are History Mode in routes in Vue.js?
How does this work?
Why there is a need for this?
What are the advantages to use it and when to use it?
How to implement this with code (also give theory explanation of code)?
You can take 2-3 examples with codes of different types to explain this.
etc....

Note:

Plagiarism is strictly prohibited.
Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.
Files need to be added into the Week7 folder only.

### Domain

Frontend Dev React/Angular/Vue

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by HastiSutaria**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #7634 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [x] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x]I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by anandhu720** [comment]
@SourabhSaraswat-191939 Please review it

**Comment by HastiSutaria** [comment]
@SourabhSaraswat-191939 PLEASE REVIEW MY PR.

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md:222]: make this all code snippets of the javascript or jsx

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md:195]: make this all code snippets of the javascript or jsx

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md:162]: make this all code snippets of the javascript or jsx

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 7/History Mode in routes.md:117]: make this all code snippets of the javascript or jsx

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@HastiSutaria make this code snippets of the jsx or javascript that's look nice

**Comment by heet-kumar** [comment]
@SourabhSaraswat-191939 review this PR once from your side bro.

**Comment by HastiSutaria** [comment]
> @HastiSutaria make this code snippets of the jsx or javascript that's look nice

Done. @heet-kumar

**Review by heet-kumar — APPROVED** [review]
@HastiSutaria Nice PR Every things looks good 👍 lets wait for the supervisor approval if they mention any changes make that to .

**Review by anandhu720 — APPROVED** [review]

**Comment by SourabhSaraswat-191939** [comment]
Good Job @HastiSutaria

**Pull request merged**
